frontend/src/routes.js

```diff
@@ -101,19 +101,19 @@ const MLStudioCreateForm = Loadable(
 );
 
 const DashboardList = Loadable(
-  lazy(() => import('./modules/Misc/views/Dashboards/DashboardList'))
+  lazy(() => import('./modules/Dashboards/views/DashboardList'))
```

Comment:
We should add `config.modules.dashboards.active` here and in `DefaultSidebar.js`
You can use `config.modules.notebooks.active` as reference

Reply:
Updated with [f84d758](https://github.com/awslabs/aws-dataall/pull/600/commits/f84d758a1514407246bb08898630ebbd07843928)